tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
should match https://github.com/senecajs/seneca-flow/blob/main/tsconfig.json

seems like the little web server is being build at the top level, but this is creating a confusing code structure

move all the web stuff into a sub folder called `web` with it's own package.json etc

Reply:
we can now change this to dist as you've moved the web stuff to `web`